### PR TITLE
Templateset.xml deployed seperately

### PR DESCRIPTION
**Pull request by Ali7223**
#1521 
I have deployed an empty xml  file for future references and also wrote Test which confirms the deployment

### CODE REVIEW

**Comment by EduardKrieger** [comment]
Change merge request branch to template-set-deployables

File: cobigen-cli/cli-systemtest/src/test/resources/testdata/localmavenproject/attachartifact.project/pom.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                  <file>${basedir}/template-set.xml</file>
+                  <type>xml</type>
+                </artifact>
+              </artifacts>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
```
Comment (EduardKrieger) [inline cobigen-cli/cli-systemtest/src/test/resources/testdata/localmavenproject/attachartifact.project/pom.xml:33]: I think this should be in the top level pom

File: cobigen-cli/cli-systemtest/pom.xml [hunk 2]
```diff
@@ -112,6 +113,25 @@
...
+              <arguments>
+                <argument>install</argument>
+                <!-- https://stackoverflow.com/a/66801171 -->
+                <argument>-Djansi.force=true</argument>
+                <argument>-Djansi.passthrough=true</argument>
+                <argument>-B</argument>
+                <argument>-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn</argument>
+              </arguments>
+            </configuration>
+          </execution>
```
Comment (EduardKrieger) [inline cobigen-cli/cli-systemtest/pom.xml:134]: Not needed in this file

File: cobigen-cli/cli-systemtest/src/test/resources/testdata/localmavenproject/attachartifact.project/pom.xml (hunk 1 above)
Comment (maybeec) [reply cobigen-cli/cli-systemtest/src/test/resources/testdata/localmavenproject/attachartifact.project/pom.xml:33]: At least on top Level pom of templates.
So far you tested test code but no change has been done on the actual application code

**Review by maybeec — CHANGES_REQUESTED** [review]
Missing fix for the actual template sets

**Comment by EduardKrieger** [comment]
PR moved to [1562](https://github.com/devonfw/cobigen/pull/1562)

**Pull request closed without merging**